Launcher/Classes/ViewControllers/InspectorViewController.swift

```diff
@@ -250,18 +231,23 @@ class InspectorViewController: NSViewController, NSTableViewDataSource {
     
     func startDevice() {
         try? fileManager.removeItem(atPath: "/tmp/screenshot_0.png")
-        let taskQueue8 = DispatchQueue.global(qos: .background)
         
-        taskQueue8.async {
-            let path = Constants.FilePaths.Bash.startDevice
-            self.runDeviceTask2 = Process()
-            self.runDeviceTask2.launchPath = path
+        if let launchPath = Constants.FilePaths.Bash.startDevice {
+            let outputStream = CommandsCore.CommandTextOutputStream()
+            outputStream.textHandler = { text in
+                if !text.isEmpty {
```

Comment:
maybe make this `guard !text.isEmpty else { return }` to prevent some indentation?